python/example_code/scheduler/scenario/README.md

```diff
@@ -0,0 +1,79 @@
+# Amazon EventBridge Scheduler Workflow
+
+## Overview
+This example shows how to use AWS SDK for Python (Boto3) to work with Amazon EventBridge Scheduler with schedules and schedule groups. The workflow demonstrates how to create and delete one-time and recurring schedules within a schedule group to generate events on a specified target, such as an Amazon Simple Notification Service (Amazon SNS) Topic.
+
+The target SNS topic and the AWS Identity and Access Management (IAM) role used with the schedules are created as part of an AWS CloudFormation stack that is deployed at the start of the workflow, and deleted when the workflow is complete.
+
+![Scheduler scenario diagram](resources/scheduler-workflow.png)
```

Comment:
This link seems incorrect, and the image is not displayed.

Reply:
good catch.